llvm/test/Transforms/InstCombine/bitcast-bfloat-half-mixing.ll

```diff
@@ -0,0 +1,70 @@
+; RUN: opt -S %s | FileCheck %s
```

Comment:
You probably want to add -passes=instcombine and generating the check lines with the update_test_checks.py script would be good. It keep the tests more standard and maintainable.

Reply:
Cool will do